### ISSUE TITLE
SSL - Blank Screen at http://mitm.it

### ISSUE THREAD

**Issue by ellecarrier** [dutzi/tamper#27]
https://github.com/dutzi/tamper/wiki/Troubleshooting Step 6

Navigating to http://mitm.it with Tamper up and running, I just get a blank screen and no options.
(Previously when tamper was not working I got the "not passing through mimproxy" message)

![screenshot 2015-05-28 14 22 26](https://cloud.githubusercontent.com/assets/2518700/7860950/141c234e-0545-11e5-8e04-baf616890b7d.png)

**Comment by ellecarrier** [comment]
Just to add, mitmproxy appears to be working ok and I've tried different port settings, still the blank screen?

**Comment by ellecarrier** [comment]
I remain stuck at troubleshooting step 6: https://github.com/dutzi/tamper/wiki/Troubleshooting
So I've moved over to using Charles, for the time being...

**Comment by saalmaan** [comment]
@ellecarrier you don't need this step to use tamper.

**Comment by ellecarrier** [comment]
Surely I do for https sites?! I'm pretty sure it doesn't work in this scenario without mitmproxy enabled?

**Comment by saalmaan** [comment]
yes you do need that to debug secure http connections, to see http://mitm.it you have to route traffic from your network card. Follow last part of step 5

> Note: (This setting is not required for Tamper to work, just to validate correct Mitmproxy installation) You can also change your network settings to see if Mitmproxy is intercepting the traffic, Open System Preferences → Network → WIFI or Ethernet → Advance → Proxies → Web Proxy (HTTP) → Web Proxy Server = 127.0.0.1:8080, save these details and apply them

**Comment by moming2k** [comment]
Open Keychain access 

Goto Certificates On the left tap

![screen shot 2015-12-14 at 10 58 21 pm](https://cloud.githubusercontent.com/assets/22466/11784259/519bc198-a2b6-11e5-930e-842f7ab9564a.png)

Double tap mitmproxy ( the one with cross )

Change the below setting from this 

![screen shot 2015-12-14 at 11 00 28 pm](https://cloud.githubusercontent.com/assets/22466/11784290/81954ad6-a2b6-11e5-9ddc-bbdf273c44b9.png)

To This ( Always Trust )

![screen shot 2015-12-14 at 11 01 16 pm](https://cloud.githubusercontent.com/assets/22466/11784314/9a21b71a-a2b6-11e5-9b73-dcb560197f81.png)

After that , restart the Chrome , re-enable the Tamper ( restart the proxy if need ) 

If it is success, it should show follow information if you expand the cert information on a SSL website. 

![screen shot 2015-12-14 at 11 03 08 pm](https://cloud.githubusercontent.com/assets/22466/11784397/fdc13534-a2b6-11e5-98ac-09b9e827b0b8.png)
